sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/service/ConfigurationServiceTest.java

```diff
@@ -83,7 +83,7 @@
             + "        shardingColumn: order_id\n";
     
     private static final String MASTER_SLAVE_RULE_YAML = "masterDataSourceName: master_ds\n" + "name: ms_ds\n" + "slaveDataSourceNames:\n" + "- slave_ds_0\n" + "- slave_ds_1\n";
-    
```

Comment:
You should modify `Keep indents on empty lines` in your IDEA's code style.

Reply:
ok，thanks!